src/js/pensions/components/DisclosureTitle.jsx

```diff
@@ -8,7 +8,7 @@ export default function createDisclosureTitle(path, title) {
     // A little hackish
     if (path === 'spouse') {
       const marriages = formData.marriages || [];
-      fullName = _.get(['marriages', marriages.length - 1, 'spouseFullName'], formData);
+      fullName = _.get(['marriages', marriages.length - 1, 'spouseFullName'], formData) || {};
```

Comment:
I don't think the `lodash/fp` version has the default value parameter (all fp versions of functions have a fixed number of parameters).